### DISCUSSION TITLE
.deb file for Debian 12 Bookworm

### DISCUSSION THREAD

**Discussion by mariovillani** [deskflow/deskflow#7819] [Q&A] [answered]
Hello,
when Deskflow was made available, i was able to download a .deb file which installed without issues on my Debian 12 machine. I'm still running that without any issues/faults. However, i no longer possess the installer and if i try installing Trixie .deb package on another computer with Debian 12, several dependencies are not met.

    dpkg: dependency problems prevent configuration of deskflow:
     deskflow depends on libc6 (>= 2.38); however:
      Version of libc6:amd64 on system is 2.36-9+deb12u8.
     deskflow depends on libei1 (>= 1.0.0); however:
      Package libei1 is not installed.
     deskflow depends on libglib2.0-0t64 (>= 2.28.0); however:
      Package libglib2.0-0t64 is not installed.
     deskflow depends on libportal1 (>= 0.8.0); however:
      Package libportal1 is not installed.
     deskflow depends on libpugixml1v5 (>= 1.4); however:
      Package libpugixml1v5 is not installed.
     deskflow depends on libqt6core6t64 (>= 6.7.2); however:
      Package libqt6core6t64 is not installed.
     deskflow depends on libqt6gui6 (>= 6.6.0); however:
      Version of libqt6gui6:amd64 on system is 6.4.2+dfsg-10.
     deskflow depends on libqt6widgets6 (>= 6.7.2); however:
      Version of libqt6widgets6:amd64 on system is 6.4.2+dfsg-10.
     deskflow depends on libssl3t64 (>= 3.0.0); however:
      Package libssl3t64 is not installed.
     deskflow depends on libstdc++6 (>= 14); however:
      Version of libstdc++6:amd64 on system is 12.2.0-14.
     deskflow depends on libtomlplusplus3t64 (>= 3.2.0); however:
      Package libtomlplusplus3t64 is not installed.

Can the Bookworm version be included again in the continuous build? Thank you in advance.

**Comment by sithlord48** [comment]
The deb package is made for debian 13+ . 
The libraries needed are not on debian 12. keep using the older version .

**Reply by mariovillani** [reply]
unfortunately i do no longer possess the .deb file. Deb file of release 1.17 for Bookworm has been overwritten with Trixie one. where can i retrieve the older version? thanks

**Comment by sithlord48** [comment]
you can build it locally or use a version of barrier / input-leap as long as you dont need wayland support.

**Comment by nbolton** [comment, marked as answer]
Try this: https://github.com/deskflow/deskflow/releases/download/v1.17.0/deskflow_1.17.0-2_debian-12_amd64.deb

Unfortunately we don't have plans to include Debian 12 in the continuous build. Please use Debian Trixie if you want to test the continuous build.

**Reply by mariovillani** [reply]
thanks! this was the fastest solution, i suggest to include this deb in the release section.

**Reply by nbolton** [reply]
Your wish is my command
https://github.com/deskflow/deskflow/releases/tag/v1.17.0
